cdap-common/bin/functions.sh

```diff
@@ -704,10 +704,13 @@ cdap_version() {
 # returns: true
 #
 cdap_sdk_usage() {
+  echo
   echo "Usage: ${0} sdk {start|stop|restart|status|usage}"
+  echo
   echo "Additional options with start, restart:"
   echo "--enable-debug [ <port> ] to connect to a debug port for Standalone CDAP (default port is 5005)"
```

Comment:
This port (`5005`) occurs in two places; should it be a variable so that it is obvious where to change it if it (the default) is changed?